massgov/mayflower
# DP-16378 Adding dangerfilejs to repository

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Adding Dangerfile to check for changelog file each time a PR is created.

Still needed:
- [ ] Add to the CircleCI config to run on all PRs except for the release branch and master branch.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-16378)
- [Github issue]()

## Steps to Test


1. CircleCI needs to run on this PR first. Below you can see that without a YAML this PR gives error from Danger.
2. Create a changelog correctly to see if works
3. Adjust the correct changelog to see if it fails from the YAML syntax checker. (Required fields type, description, component name, project prefix) See the changelog template which was updated.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Just know when you switch back to develop you will see the node_modules appear you can either reset or we can update the develop .gitignore before this PR goes out.

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
